contracts/ERC23.sol

```diff
@@ -0,0 +1,7 @@
+pragma solidity ^0.4.11;
```

Comment:
Is it still a problem? I think a separate PR with just this change should be better